src/apiCalls.js

```diff
@@ -19,12 +21,32 @@ export const fetchSingleMovie = (id) => {
 }
 
 export const fetchMovieTrailers = (id) => {
-  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${+id}/videos`)
+  return fetch(`${endpoint}/movies/${+id}/videos`)
   .then(response => response.json());
 }
 
-// export {
-//   fetchMovies,
-//   fetchSingleMovie,
-//   fetchTrailers
-// };
\ No newline at end of file
+export const postLoginCredentials = (loginCredentials) => {
+  return fetch(`${endpoint}/login`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    }, 
+    body: JSON.stringify(loginCredentials)
+  })
+    .then((response) => {
+      if (!response.ok) {
+        throw Error('Username and/or password is incorrect. Please try again.')
+      }
+      return response.json()
+    })
```

Comment:
We are repeating this line of code in every statement - let's refactor this into another function.